build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -192,6 +192,7 @@ function(iree_run_module_test)
     WILL_FAIL
       ${_TEST_XFAIL}
     LABELS
+      "test-type=run-module-test"
```

Comment:
Nice, I like this :-) Maybe we should put similar labels on our other test rules.